projects/gateway2/api/v1alpha1/kube_types.go

```diff
@@ -96,6 +96,11 @@ type Service struct {
 	//
 	// +kubebuilder:validation:Optional
 	ExtraAnnotations map[string]string `json:"extraAnnotations,omitempty"`
+
+	// External Traffic Policy on the Service object.
+	//
+	// +kubebuilder:validation:Optional
+	ExternalTrafficPolicy *corev1.ServiceExternalTrafficPolicy `json:"externalTrafficPolicy,omitempty"`
```

Comment:
Consider setting the default value of this field for [CRD validation](https://book.kubebuilder.io/reference/markers/crd-validation) purposes:
```go
// +kubebuilder:default=Cluster
```

Reply:
In general we want default values to be explicitly represented in our APIs ([xref](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#defaulting)).